### ISSUE TITLE
nodejs LTS version old

### ISSUE THREAD

**Issue by chhschou** [ubuntu/ubuntu-make#434]
Fresh install of Ubuntu 16.04 LTS. Installed umake version 17.03. Then installed nodejs with --lts. nodejs version installed is v4.8.1. I would expect at least v6.9.1.

How would I go about updating this through umake?

**Comment by LyzardKing** [comment]
The url code that I thought was generic for the lts releases has been
changed. I'll try and change it later today if I can.

**Comment by LyzardKing** [comment]
@chhschou It's fixed with #435 
It'll be added in the next release. For now you can run from the source tree or build a snap.

**Comment by chhschou** [comment]
Apology for taking so long. I also had a bit of issue figuring out how to work with the source to try the new source. I manage to download and build and install via snap. However the nodejs version installed is still v4.8.1. I see the SHA256 it is trying to use is: https://nodejs.org/dist/latest-argon/SHASUM256.txt

As a next step t I will go check now whether the source I downloaded has your changes.

**Comment by chhschou** [comment]
Okay, I found issue with building my snap, the latest code works and installs v6.10.1 LTS. 
TL;DR:
Bizarre after I moved the `snapcraft.yml` file to root directory of source it build with the new code, it seems it was pulling and building with old code previously or something. However I then did `snapcraft clean` then move the `snapcraft.yml` back under the `snap` folder and call `snapcraft` at the root directory again and it also builds the latest (I inserted a couple of debug `print` statements in code which was displayed accordingly).
